src/batch.rs

```diff
@@ -50,6 +50,15 @@ impl Batch {
         let (max_size_value_length, max_size_unit_length) = self.detect_size_lenghts();
 
         for meta in &self.0 {
+            let mut size_str;
+            if let Some(ref size) = meta.size {
+                size_str = size.render(colors, max_size_value_length, max_size_unit_length);
+            } else {
+                size_str = ANSIString::from(
+                    " ".repeat(max_size_value_length + max_size_unit_length) + "-",
+                );
```

Comment:
I'm not really enthusiastic about adding some Size related logic inside the Batch struct. The Symlink logic just above is a bad example and should be moved into the Symlink.render() method.